frontend/package.json

```diff
@@ -205,7 +205,7 @@
     "react-i18next": "^11.12.0",
     "react-linkify": "^0.2.2",
     "react-modal": "^3.16.3",
-    "react-redux": "7.2.9",
+    "react-redux": "8.1.3",
```

Comment:
Note that `react-redux` is Console provided shared module, using the default config
```js
{ singleton = true, allowFallback = false }
```
i.e. plugins are built with the assumption that Console provides v7 impl. without plugin provided fallback.

So this is effectively a breaking change in Console provided shared modules.